packages/e2e-test-utils/src/canvas.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Gets the editor canvas frame.
+ */
+export function canvas() {
```

Comment:
Not sure the reason behind naming it this way was worthwhile.

Seeing it used in a test as `canvas()`, it looks really bizarre.

Reply:
Any suggestions? This will be used a lot in all tests when the iframe will be used for the post editor, so it would be good if it's short like `page` so it doesn't trigger code reformatting for thousands of lines.